### DISCUSSION TITLE
데이터베이스 정리

### DISCUSSION THREAD

**Discussion by Sir-u** [Sir-u/InternetDB-Project#13] [Announcements]
회원가입에 필요한 항목

**Comment by huichangs** [comment]
"STDNUM" NUMBER(10,0) NOT NULL ENABLE, 
	"PASSWD" VARCHAR2(100 BYTE), 
	"USRNAME" VARCHAR2(20 BYTE), 
	"GENDER" VARCHAR2(20 BYTE), 
	"BIRTHDAY" VARCHAR2(20 BYTE), 
	"AGE" NUMBER(10,0), 
	"REG_DATE" VARCHAR2(20 BYTE)

**Comment by huichangs** [comment]
"STDNUM" NUMBER(9,0) NOT NULL ENABLE, 
	"PASSWD" VARCHAR2(100 BYTE), 
	"USRNAME" VARCHAR2(20 BYTE), 
	"GENDER" VARCHAR2(20 BYTE), 
	"BIRTHDAY" VARCHAR2(20 BYTE), 
	"AGE" NUMBER(10,0), 
	"REG_DATE" VARCHAR2(20 BYTE), 
	"AUTHORITY_LEVEL" NUMBER(1,0) DEFAULT 0, 
	 PRIMARY KEY ("STDNUM")
권한별 접근 구현으로 인해 데이터베이스 column에 authority_level 항목이 추가되었습니다. 
해당 항목은 따로 입력 받는 칸이 필요 없습니다. 따라서 변경할 부분 또한 없습니다. 
참고용으로만 확인해 주시면 되겠습니다.

**Reply by siwhan99** [reply]
reply테이블입니다("REPLY_NUM" NUMBER, 
	"REPLY_USERNAME" VARCHAR2(100 BYTE), 
	"REPLY_CONTENT" VARCHAR2(1000 BYTE), 
	"REPLY_TIME" TIMESTAMP (6), 
	 FOREIGN KEY ("REPLY_NUM")
	  REFERENCES "SCOTT"."BOARD" ("NUM") ENABLE)

board테이블"NUM" NUMBER, 
	"USERNAME" VARCHAR2(50 BYTE), 
	"PASSWORD" VARCHAR2(50 BYTE), 
	"TITLE" VARCHAR2(100 BYTE), 
	"MEMO" VARCHAR2(4000 BYTE), 
	"TIME" TIMESTAMP (6), 
	"HIT" NUMBER, 
	 CONSTRAINT "PK_BOARD" PRIMARY KEY ("NUM") 입니다 .
